Juggernaut9/pokemon
# Sort Pokemon list as per their ids

How to test?

1. Checkout branch
2. Run `npm start`.
3. The pokemons should come sorted by their ids. Please verify the UI from the `pokemon.json` file.
